apps/sse_cfg/src/impl_prog/prog_api.erl

```
@@ -704,20 +706,22 @@ transaction(F) ->
         {reason, Reason}]),
       {error, Reason};
 
-    % Catches {ok, Result} = foo() with error return.
+    % Catches {ok, Result} = foo() with error return and stack.
     {aborted, {{Class, {error, Reason}}, Stack}} ->
       ?ERROR([
         {class, Class},
         {error, transaction_aborted},
         {reason, Reason},
         {stack, Stack}]),
-      {error, Reason};
+      {error, Reason}
+  catch
```

There is a behaviour change as if `mnesia:transaction(F)` returns unknown value then a `try_clause` clause is thrown rather than logging `?ERROR`.

From https://erlang.org/doc/reference_manual/expressions.html#try :
> Exceptions ... due to a failed match are not caught.